cocos/3d/CCMesh.cpp

```diff
@@ -350,6 +350,7 @@ void Mesh::draw(Renderer* renderer, float globalZOrder, const Mat4& transform, u
 
 
     _meshCommand.setSkipBatching(isTransparent);
+    _meshCommand.setTransparent(isTransparent);
```

Comment:
Bug fix. When we render the transparent navmesh, we find there is something wrong.